### PR TITLE
Region Availability: Adding SDK changes for Upcoming regions

### PR DESCRIPTION
**Pull request by ashutoast7**
This PR makes several regions available for public usage.

Belgium Central
Denmark East
Israel Northwest
Southeast US 3
Southeast US 5

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Include samples if adding new API, and include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

**Review by kirankumarkolli — DISMISSED** [review]

**Comment by ashutoast7** [comment]
@microsoft-github-policy-service agree company="Aptly Technology Corporation"

**Review by FabianMeiswinkel — DISMISSED** [review]
LGTM

File: Directory.Build.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		<ClientOfficialVersion>3.46.0</ClientOfficialVersion>
 		<ClientPreviewVersion>3.47.0</ClientPreviewVersion>
 		<ClientPreviewSuffixVersion>preview.0</ClientPreviewSuffixVersion>
-		<DirectVersion>3.37.3</DirectVersion>
+		<DirectVersion>3.37.4</DirectVersion>
```
Comment (simplynaveen20) [inline Directory.Build.props:6]: As discussed offline, due to incorrect SOP step, 3.37.4 missed one PR change on Proxity, lets release 3.37.5 and then update here.

Comment (ashutoast7) [reply Directory.Build.props]: updated with changes, to 3.37.5

**Review by kirankumarkolli — APPROVED** [review]

**Comment by simplynaveen20** [comment]
"/azp run

**Review by simplynaveen20 — APPROVED** [review]

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Pull request merged**